tools/PI/DevHome.PI/Controls/AddToolControl.xaml

```diff
@@ -17,89 +19,123 @@
             <RowDefinition Height="Auto"/>
         </Grid.RowDefinitions>
 
-        <StackPanel Orientation="Vertical">
+        <Grid>
+            <Grid.RowDefinitions>
+                <RowDefinition Height="Auto"/>
+                <RowDefinition Height="Auto"/>
+                <RowDefinition Height="Auto"/>
+                <RowDefinition Height="Auto"/>
+                <RowDefinition Height="Auto"/>
+                <RowDefinition Height="Auto"/>
+                <RowDefinition Height="Auto"/>
+            </Grid.RowDefinitions>
+
             <Grid>
+                <Grid.ColumnDefinitions>
+                    <ColumnDefinition Width="100"/>
+                    <ColumnDefinition Width="*"/>
+                </Grid.ColumnDefinitions>
+                <Button
+                    x:Name="AppListButton" x:Uid="AppListButton" HorizontalAlignment="Left" VerticalAlignment="Top" MinWidth="100" Click="AppListButton_Click"/>
+                <controls:DataGrid
```

Comment:
Why did you choose to use a DataGrid?  These controls are heavy and might not meet accessibility standards.

Reply:
You're right, thanks. I was toying with adding a few more columns, and DataGrid is good for column sorting/re-arranging/etc - but in the end I kept it down to 2 columns. Updated to use a simple ListView.